lib/govuk_index/presenters/common_fields_presenter.rb

```diff
@@ -53,6 +53,7 @@ def popularity
     end
 
     def format
+      return 'smart-answer' if payload['publishing_app'] == 'smartanswers'
```

Comment:
Would it be possible to use something like the schema instead? I'm worried that if the publishing app ever changed, it would silently (and mysteriously) break search.

Reply:
I think one is as likely to change as the other. I will check with @MatMoore what he thinks the best field to use for this check would be.

Reply:
also just as a point, this field is useful now for the migration, however once that is complete it wont be used/required so this line could then be removed.